src/sage/rings/integer.pyx

```diff
@@ -2951,13 +2951,15 @@ cdef class Integer(sage.structure.element.EuclideanDomainElement):
 
         return n
 
-    def prime_divisors(self):
+    def prime_divisors(self, *, limit=None):
         """
         Return the prime divisors of this integer, sorted in increasing order.
 
         If this integer is negative, we do *not* include -1 among
         its prime divisors, since -1 is not a prime number.
 
+        If ``limit`` is given, only primes up to the limit are returned.
```

Comment:
This doesn't follow the format for documentation; I would expect something along the lines:
https://github.com/sagemath/sage/blob/0f06092e1d7196f372505600d7459b757cd21d63/src/sage/rings/integer.pyx#L3824-L3854

And perhaps, why not pass all the optional arguments?

Reply:
> And perhaps, why not pass all the optional arguments?

I just tried it, but this alone doesn't quite work as it should: `Integer.factor()` often finds larger prime divisors than the bound, and of course the `Factorization` also includes the unfactored rest. So it still needs special treatment for `limit=`.

Reply:
you could do 
```
def prime_divisors(self, **kwargs):
...
    res = [r[0] for r in self.factor(**kwargs)
    limit = kwargs.get('limit')
    if limit is not None:
...
```

Reply:
Thanks for the suggestion — done.